xbmc/windowing/osx/OpenGL/OSXGLView.mm

```diff
@@ -72,6 +86,7 @@ - (void)drawRect:(NSRect)rect
     glClearColor(0, 0, 0, 0);
 
     [m_glcontext update];
+
```

Comment:
what is this for? :)

Reply:
:) I'll remove before merging. Thanks for the review